### PR TITLE
Prevent failed build checks due to labeler

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -16,3 +16,4 @@ jobs:
       with:
         sync-labels: ""
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
+      continue-on-error: true
```
Comment (sungam3r) [inline .github/workflows/label.yml:19]: Was a labeler error once an obstacle for other actions? See #2094:
изображение

**Comment by Shane32** [comment]
An obstacle? Not that I know of.

**Comment by Shane32** [comment]
This really bugs me:

image

**Comment by sungam3r** [comment]
Now I understand.
изображение
vs
изображение

Maybe switch to the second? In fact, the result is the same, but it seems to me that the intention will be expressed more clearly.

**Review by sungam3r — APPROVED** [review]

**Pull request merged**